pkg/ui/workspaces/cluster-ui/src/databases/util.tsx

```diff
@@ -102,7 +142,7 @@ export function nodesByRegionMapToString(
     .join(", ");
 }
 
-export function createNodesByRegionMap(
+function createNodesByRegionMap(
```

Comment:
`nodesByRegionMapToString` and `createNodesByRegionMap` are only used in this file, so I removed the exports.